### DISCUSSION TITLE
Requesting Units for Dimensions?

### DISCUSSION THREAD

**Discussion by philsmy** [hakanensari/peddler#163] [Q&A]
We are trying to pull down item dimensions for European Marketplaces
In the GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA report things come down correctly in cm:

“longest-side”: “15.0”,
“median-side”: “14.0”,
“shortest-side”: “8.0”,
“length-and-girth”: “59.0”,
“unit-of-dimension”: “centimeters”

But in the MWS calls like GetProductInfoById it comes down in inches!
{“Height”:{“content”:“3.1496062960”, “Units”:“inches”},
“Length”:{“content”:“5.9055118050”, “Units”:“inches”},
“Width”:{“content”:“5.5118110180”, “Units”:“inches”},
“Weight”:{“content”:“0.454152259720”, “Units”:“pounds”}},

It’s much easier for our requirements to use the MWS api, but obviously these measurements are ridiculous. Is there no way to set the units for the MWS calls?

**Comment by hakanensari** [comment]
It's strange that the European endpoint doesn't default to metric. In any case, unless I am missing something in the docs, there doesn't seem to be a way to set the unit.

Have you tried reaching out to their support? Perhaps there's an undocumented way to do this.
